src/main/scala/SentimentAnalysis.scala

```diff
@@ -1,27 +1,63 @@
-import com.azure.core.credential.AzureKeyCredential
-import com.azure.ai.textanalytics.models._
-import com.azure.ai.textanalytics.TextAnalyticsClientBuilder
-import com.azure.ai.textanalytics.TextAnalyticsClient
-        
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.functions.udf
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.expressions._
+import org.apache.spark.sql.types._
+import java.io.FileNotFoundException
+import scala.collection.mutable.ListBuffer
+import org.apache.spark.sql.DataFrame
+
 object SentimentAnalysis{
-    final val KEY: String = "da18b5be9da74415b8118936d39f3f49"
-    final val ENDPOINT: String = "https://twitteranalysisproject.cognitiveservices.azure.com/"
-    final val client = authenticateClient(KEY, ENDPOINT);
-    
 
-    def authenticateClient(key: String, endpoint: String): TextAnalyticsClient = {
+      def sentimentAnalysis(text: String): String = {
+        val spark = SparkSession.builder().appName("Sentiment Analysis").getOrCreate()
+        import spark.implicits._
+        //Read a text file and convert it into Dataframe
+        val sentimentLexicon = spark.read
+            .option("inferSchema", "true")
+            .textFile("/datalake/Subjectivity_Lexicon.txt").cache()
 
-    return new TextAnalyticsClientBuilder()
-        .credential(new AzureKeyCredential(key))
-        .endpoint(endpoint)
-        .buildClient();
-    }
+        val lexiconToArray = sentimentLexicon.collectAsList.toArray
+        var mappingWordAndSentiment = ListBuffer[Map[String, String]]()
+        for(i <- lexiconToArray){
+            val words = i.toString.split(" ")
+            var word1 = ""
+            var sentiment1 = ""
+            for(j <- words){
+                if(j.contains("word1")) {
+                    word1 = j.split("=")(1)
+                }else if(j.contains("priorpolarity")) {
+                    sentiment1 = j.split("=")(1)
+                }
+            }
+            mappingWordAndSentiment += Map(word1 -> sentiment1)
+        }
```

Comment:
to my understanding the above code is stripping Subjectivity_Lexicon.txt to a usable format, can you put a short comment explaining what its starting as and what you want it to be

Reply:
`val lexiconToArray = sentimentLexicon.collectAsList.toArray` //I'm deconstructing Lexicon DF into Array becuase it contains too many info.

    `var mappingWordAndSentiment = ListBuffer[Map[String, String]]()`   //After grabbing only a "Word" and its "Sentiment" from the lexicon, I will put them back in ListBuffer in a Map format


      ```
       for(i <- lexiconToArray){                      //reading each line from Lexicon
            val words = i.toString.split(" ")       //splitting each line by space, and getting words in the line
            var word1 = ""                                  //after grabbing a "Word", I store it here to map it later
            var sentiment1 = ""                         //after grabbing a "Sentiment", I store it here to map it later
            for(j <- words){                               //reading word by word in each line
                if(j.contains("word1")) {             //the word is in this format: "word1=abhor". So, I'm checking if this is grabbing a word
                    word1 = j.split("=")(1)            //after confirming it is a word, I store it to word1
                }else if(j.contains("priorpolarity")) {  //the sentiment is in this format: "priorpolarity=negative". So, I'm checking if this is grabbing a sentiment
                    sentiment1 = j.split("=")(1)    //after confirming it is a sentiment, I store it to sentiment1
                }
            }
            mappingWordAndSentiment += Map(word1 -> sentiment1)  //After mapping a word with its sentiment, I'm storing them in ListBuffer
        }
        ```